tests/Aspire.Hosting.Tests/ManifestGenerationTests.cs

```diff
@@ -114,6 +114,46 @@ public void EnsureExecutablesWithDockerfileProduceDockerfilev0Manifest()
         Assert.Equal("{nodeapp.bindings.https.port}", env.GetProperty("HTTPS_PORT").GetString());
     }
 
+    [Fact]
+    public void EnsureExecutablesWithDockerfileAndBuildArgsProduceDockerfilev0Manifest()
```

Comment:
Move this test to container test and make it use the new way of validating the manifest (as a giant string)

Reply:
Should all of these tests be moved then? I admittedly, just copied and pasted an existing test, and added a few bits.

Reply:
New ones first. Old ones will get renovated. We’re not adding new ones to this file.

Reply:
I just removed this test, as it's redundant with the other tests I have in place.

Reply:
We should add a comment to that effect